### ISSUE TITLE
Unexpected and undesired changes into the database

### ISSUE THREAD

**Issue by gasbi** [wix-incubator/corvid#75]
**Describe the bug**
When using the local editor `npx corvid open-editor`, I notice that the files in `src/database/*` are changed even though I did not do any change into the database. And these changes are breaking down my database schema.

**To Reproduce**
1. Clone wix site with corvid
2. Open editor with corvid
3. Do not do any change on the site.
4. Click on `Local Save`.
3. Close editor (without changing anything in the database).
4. Run `git status` and you can see undesired changes in `src/database`.

**Expected behavior**
I would expect some changes in some files due to "behind the scenes" changes from the editor, **but never a change on the database schema** if I haven't done that.

**Screenshots**
In this screenshot, it can be observed how this affected to my database schema, and I am certain that I didn't do any of those changes.
![image](https://user-images.githubusercontent.com/10176387/60323288-eeef4580-9981-11e9-8728-a56c639844e3.png)

**Desktop (please complete the following information):**
 - OS: macOS Mojave 10.14.5 (18F132)
 - Version: Corvid CLI v0.1.76
- Site URL: https://eparelllc.wixsite.com/mysite

**Additional context**
We are aware Corvid CLI is in Alpha, and would appreciate any help the Corvid team can provide.

Thanks very much!

**Comment by dor-itzhaki** [comment]
Thanks for the details description @gasbi.
Can you please also post the URL of the wix site (public or editor) where you see this reproduced ?

**Comment by gasbi** [comment]
Hello @dor-itzhaki, thanks for the reply!

The URL of the published site is: https://eparelllc.wixsite.com/mysite

Let me know of any additional help I can provide 👍

**Comment by gasbi** [comment]
Re-checking the bug, I have found better instructions to reproduce it: you need to click on `local save` button, even though you have not done any change to the site.

Those are updated in the issue description now.

**Comment by stevesos** [comment]
Here's an image showing Visual Studio Code in the background with the rogue files in yellow (indicating the Git changes) You can see that the dynamic page list has empty slots between DefaultAssortment and DefaultPosition, DefaultPosition and Department, and after Outfit. These correspond nicely with the Yellow files in the background ;-).

![image](https://user-images.githubusercontent.com/49377482/60598498-b649d480-9da4-11e9-984b-028f13571069.png)

We have determined that the problem is with the local editor "synchronizing" schema files with the online Wix Editor and wrongly downloading missing files from the current online version. 

Bottom line the local editor SHOULD NOT even look at the online site files except when it performs a Push.

**Comment by tomenden** [comment]
Hi @gasbi,

Thank you very much for reporting the issue.
The reason for the issue is **multi-reference** fields which are currently being used in your site. Currently, `corvid-cli` does not support multi-reference. We are working on resolving the issue, and I expect it to be resolved next week.

**Please Note:**
Features relating to the Local Editor are currently in the stage of alpha release and are still being developed. Future functionality may not be backward compatible. When using these features, you assume the risks associated with the use of such an alpha version.

**Comment by gasbi** [comment]
Hi @tomenden,

Glad to help! 
Thank you very much for the reply and the estimated timeline. That's very helpful for us.
This issue together with #85 are holding our development on the local editor, will it be possible to get an ETA on that one as well?

Thanks again!

**Comment by dor-itzhaki** [comment]
The issue should now be solved (it might take a bit until the fix is rolled out to all users)

**Comment by gasbi** [comment]
Thanks @dor-itzhaki! Is it required to update the npm corvid-cli package to see the changes?

**Comment by dor-itzhaki** [comment]
> Thanks @dor-itzhaki! Is it required to update the npm corvid-cli package to see the changes?

nope. it's the online wix editor that was modified.

**Comment by gasbi** [comment]
Hi @tomenden and @dor-itzhaki,

I have been testing the corvid-cli and this issue seems to be solved at a major extent, but I am still getting some changes on the database. These are my results:

**Solved:**
- There is no more deletion of fields.
- The field `displayName` does not disapper.

**Still experiencing:**
- Changes on the `index` field, which I suppose is not of great importance.
- Changes on the `sortable` field, which seems to be more important.

Are these changes something one would expect? or are they known issues?

Here a screenshot for clarification:
<img width="952" alt="Screenshot 2019-07-24 at 11 40 35" src="https://user-images.githubusercontent.com/10176387/61783324-e31f6380-ae07-11e9-8418-baed85f699fd.png">


Thanks again!

**Comment by dor-itzhaki** [comment]
Hey @gasbi,

A few questions:

1. Do you see the same happening if you clone the site again "from scratch" and open the local editor ?
2. Do you always get the same changes or different ones each time you open the local editor ?
3. Were these changes to the schemas actually done manually by someone at some point ?

**Comment by gasbi** [comment]
Hi @dor-itzhaki,
Apologies for the late response.

At some point I stopped getting those changes, so I can say it is resolved now.

In case it's helpful, I'll answer your questions:
1. I didn't try that.
2. Yes.
3. No, actually I wouldn't know how to change the `sortable` property from the editor.

Thanks!
